client/trino-cli/src/test/java/io/trino/cli/lexer/TestStatementSplitter.java

```diff
@@ -20,9 +20,6 @@
 import static io.trino.cli.lexer.StatementSplitter.isEmptyStatement;
 import static io.trino.cli.lexer.StatementSplitter.squeezeStatement;
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
@hashhar I've tried to add a rule to `./.mvn/modernizer/violations.xml`
```
    <violation>
        <name>org/junit/jupiter/api/Assertions</name>
        <version>1.1</version>
        <comment>Prefer org.assertj.core.api.Assertions</comment>
    </violation>
```
but it seems does not work.
This is similar to already existing 
```
    <violation>
        <name>org/testng/annotations/AfterTest</name>
        <version>1.8</version>
        <comment>Prefer org.testng.annotations.AfterClass</comment>
    </violation>
```
or https://github.com/gaul/modernizer-maven-plugin/blob/80cca42449ae0e916c6e9972ee447b2e80725700/modernizer-maven-plugin/src/main/resources/modernizer.xml#L14
```
  <violation>
    <name>com/google/common/base/Predicate</name>
    <version>8</version>
    <comment>Prefer java.util.function.Predicate</comment>
  </violation>
```
 Do you have any idea how to add it properly?

Reply:
It's possible to use `maven-enforcer-plugin` instead of `modernizer`